### DISCUSSION TITLE
智能计算系统 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#203] [Announcements]
# 智能计算系统 - CS自学指南

CS自学指南

https://csdiy.wiki/%E4%BA%BA%E5%B7%A5%E6%99%BA%E8%83%BD/CYJ/

**Comment by dragonbra** [comment]
你好，实验手册的链接挂掉了

**Comment by sarendis56** [comment]
有人一起刷这个吗？

**Reply by GaoXiangYa** [reply]
有的，正在刷这个，但是新版的实验平台进不去啊

**Reply by sarendis56** [reply]
> 有的，正在刷这个，但是新版的实验平台进不去啊

你好，你说的是哪个新的实验平台？

**Reply by GaoXiangYa** [reply]
是南京麒麟的那一个

**Reply by GaoXiangYa** [reply]
大佬，你能正常做实验吗，我这边写Lab，寒武纪的实验平台传送数据集总是传不上去

**Reply by sarendis56** [reply]
你好，我没有在国内，打不开南京智能计算中心，看了一下你的issue，校外同学应该是不提供算力资源？请问你现在是用什么平台在做了

**Reply by GaoXiangYa** [reply]
我现在还是在寒武纪的实验平台上，但是寒武纪实验平台无法传送1MB以上的文件，导致我lab2的数据集没办法传上去，现在暂时卡在这了

**Reply by bitkira** [reply]
话说这个课程的书和b站更新是不是都没完成呀（2024的）

**Reply by GaoXiangYa** [reply]
@bitkira 2024版的目前还没有更新

**Comment by Just-lose-it** [comment]
请问如何登录新版算力平台啊，账号密码都没有

**Reply by lkhlzcxv** [reply]
没账号就用不了

**Comment by EnzoDing-rgb** [comment]
值得学吗
